### PR TITLE
Rework site documentation

### PR DESCRIPTION
**Pull request by Bananeweizen**
* fix wording and typos
* enable syntax highlighting
* replace tabs in code blocks for unified layout
* upgrade skin (dependabot doesn't work for site.xml)

Be aware numbered lists don't work well here, if they have code blocks in between, because the index resets to 1 after each block. Therefore numbered lists have been replaced by bullet lists several times. An alternative might be converting to Asciidoc, which has far greater control over such things.

### CODE REVIEW

File: src/site/markdown/BndBuild.md [hunk 1]
```diff
@@ -15,37 +15,34 @@ A BND Workspace layout build layout usually has the following structure:
...
+      <version>${tycho-version}</version>
+   </extension>
 </extensions>
 ```
 
-3. create a file called `maven.config` in the `.mvn` folder with the following content (adjust the version accordingly!):
-
+- create a file called `maven.config` in the `.mvn` folder with the following content (adjust the version accordingly!):
+```properties
+-Dtycho-version=4.10.0
```
Comment (mlehmannm) [inline src/site/markdown/BndBuild.md:39]: 4.10.0?

**Comment by laeubi** [comment]
@Bananeweizen can you share a screenshot of the new layout/skin?

**Comment by Bananeweizen** [comment]
@laeubi The skin itself is the exact same. This shows left=PR, right=online Tycho documentation. I've marked some areas with notable differences.
grafik

**Comment by laeubi** [comment]
@Bananeweizen Should this backported to Tycho 4?

**Comment by Bananeweizen** [comment]
@laeubi might be useful to be backported so it goes live on the website. I guess a real 5.0 release is still far away, right? but I'm also okay with not backporting, if that creates additional effort.

**Pull request merged**